src/AutoRest.CSharp/Common/Input/InputTypes/InputOperation.cs

```diff
@@ -23,7 +23,8 @@ public InputOperation(
     IReadOnlyList<InputParameter> parameters,
     IReadOnlyList<OperationResponse> responses,
     RequestMethod httpMethod,
-    BodyMediaType requestBodyMediaType,
+    // Obsolete, for swagger input only
```

Comment:
In general, we don't keep obsolete property for swagger input only.
We need to figure out the corresponding representation in tsp, and convert the information from swagger to it.

Reply:
I agree but we could take things in steps.
I imagine TCGC should already have similar replacement, but we could keep it for now and figure it out in the future to reduce the amount of changes in this PR.
The goal of this PR is to keep all the regen unchanged, and keep these legacy things could help for now